src/main/java/com/autotune/analyzer/services/CreateExperiment.java

```diff
@@ -96,8 +98,34 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             } else {
                 List<KruizeObject> kruizeExpList = new ArrayList<>();
                 for (CreateExperimentAPIObject createExperimentAPIObject : createExperimentAPIObjects) {
+                    String experimentType = createExperimentAPIObject.getExperimentType();
```

Comment:
Look for createExperimentAPIObject file commet implement interface and use those functions here

Reply:
Updated here as well.